src/helpers/convert-date.js

```diff
@@ -175,6 +175,17 @@ const formatDateToDayMonthYearHour = (date) => {
   }).format(new Date(date))
 }
 
+const getCurrentDateTimeIntl = () => {
+  return new Intl.DateTimeFormat('pt-BR', {
+    day: '2-digit',
+    month: '2-digit',
+    year: 'numeric',
+    hour: '2-digit',
+    minute: '2-digit',
+    hour12: false
+  }).format(new Date())
+}
```

Comment:
```suggestion
const getCurrentDateTimeIntl = (date) => {
  return new Intl.DateTimeFormat('pt-BR', {
    day: '2-digit',
    month: '2-digit',
    year: 'numeric',
    hour: '2-digit',
    minute: '2-digit',
    hour12: false
  }).format(date)
}
```
Recomendo alterar o código para formatar uma `Date` e retornar no formato desejado